main.go

```diff
@@ -82,6 +94,20 @@ func runPrometheusServer(metricsAddr string) {
 	}()
 }
 
+func getWebhookServer(sspTLSOptions common.SSPTLSOptions) *webhook.Server {
+	if sspTLSOptions.IsEmpty() {
```

Comment:
What is the default behavior of other parts of SSP when there is no TLS configuration set? Should it run without TLS or not at all? At least all endpoints should behave the same. IIUC the template validator and the prometheus endpoint start with empty TLS config values while the SSP webhook server will refuse to run?

Reply:
we decided it will all run if the tls config is not provided. If the webhookserver is set to nil the manager will crate a default one.

Reply:
Ah, I forgot that an `nil` webhookserver creates a default one. So everything starts with some default TLS configuration values? That's fine.